### PR TITLE
Dyn 41 : GetItem API implementation and test cases

### PR DESCRIPTION
**Pull request by ankitmalikg2**
Implement & test Get item API

### CODE REVIEW

File: service/spanner/spanner.go [hunk 1]
```diff
@@ -46,21 +44,15 @@ func ParseDDL(updateDB bool) error {
...
-				tmpCols = []string{}
-				currentTable = tableName
 			}
-			tmpCols = append(tmpCols, column)
+			models.TableColumnMap[tableName] = append(models.TableColumnMap[tableName], column)
 			models.TableDDL[tableName][column] = dataType
 		}
-		models.TableColumnMap[currentTable] = tmpCols
 	}
-
```
Comment (rosspatil) [inline service/spanner/spanner.go:63]: Why did you change it? any reason? @ankitmalikg2

Comment (ankitmalikg2) [reply service/spanner/spanner.go:63]: yes because it was working fine if we have all the columns info of a table stored continuously but when we use Spanner emulator it doesn't maintain order so need to change it so that it can work if there is an order or not.

**Review by rosspatil — APPROVED** [review]
LGTM!
